charts/tools/templates/provisioner/deployerRoleBinding.yaml

```diff
@@ -0,0 +1,19 @@
+{{ if .Values.provisioner.serviceAccount.enabled }}
+
```

Comment:
same thing as above. Potentially rename file to `provisionerRoleBinding` to not be confused with the `deployer/deployerRoleBinding` template?

Reply:
Updated to `provisionerRoleBinding` thanks!